script/results.html

```diff
@@ -55,14 +104,46 @@
       <table class="table table-hover table-bordered" id="nodes-table">
       </table>
     </div>
-    <div class="tab-pane fade" id="results" role="tabpanel" aria-labelledby="results-tab">
-      <table class="table table-hover table-bordered" id="results-table">
+
+    <div class="tab-pane fade" id="results" role="tabpanel" aria-labelledby="result-tab">
+      <table class="table table-hover table-bordered" id="results-table" hidden="hidden">
+      </table>
+      <table class="table table-hover table-bordered" id="results-far-edge-table" hidden="hidden">
+      </table>
+      <table class="table table-hover table-bordered" id="optional-table" hidden="hidden">
+      </table>
+      <table class="table table-hover table-bordered" id="results-telco-table" hidden="hidden">
+      </table>
+      <table class="table table-hover table-bordered" id="optional-table-telco" hidden="hidden">
+      </table>
+      <table class="table table-hover table-bordered" id="results-NonTelco-table" hidden="hidden">
+      </table>
+      <table class="table table-hover table-bordered" id="optional-table-non-telco" hidden="hidden">
       </table>
     </div>
+
+
     <div class="tab-pane fade" id="versions" role="tabpanel" aria-labelledby="versions-tab">
       <table class="table table-hover table-bordered" id="versions-table">
       </table>
     </div>
+    <div class="tab-pane fade" id="feedback" role="tabpanel" aria-labelledby="feedback-tab">
+      <table class="table table-hover table-bordered" id="feedback-table-faredge" hidden="hidden">
+      </table>
+      <table class="table table-hover table-bordered" id="feedback-table-non-telco" hidden="hidden">
+      </table>
+      <table class="table table-hover table-bordered" id="feedback-table-telco" hidden="hidden">
+      </table>
+      <table class="table table-hover table-bordered" id="feedback-all" hidden="hidden">
+      </table>
+      <table class="table table-hover table-bordered" id="optional-table-telco-feedback" hidden="hidden">
+      </table>
+    </div>
+    <table class="table table-hover table-bordered" id="optional-table-non-telco-feedback" hidden="hidden">
+    </table>
+    <table class="table table-hover table-bordered" id="optional-faredge-feedback" hidden="hidden">
+    </table>
+  </div>
   </div>
 
   <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js"
```

Comment:
add these lines for duration conversion:  
<script src="https://cdn.jsdelivr.net/npm/dayjs@1.10.4/dayjs.min.js"></script>
  <script src="https://cdn.jsdelivr.net/npm/dayjs@1.10.4/plugin/duration.min.js"></script>